apps/webcert/src/feature/certificate/UnifiedEdit/UeCheckboxDateRangeList/PreviousPeriod.test.tsx

```diff
@@ -0,0 +1,23 @@
+import { render, screen } from '@testing-library/react'
+import { PreviousPeriod } from './PreviousPeriod'
+
+const renderComponent = (previousSickLeavePeriod: string) => {
```

Comment:
Should we try to remove the term SickLeave and make the component more general?

Reply:
Changing the backend as well? previousSickLeavePeriod -> previousPeriod

Reply:
That could be done aswell but here I was talking about the component. It's not connected to the object used in the backend so it should be able to make general with previousPeriod without updating the backend.

Reply:
Alright, I changed the property for the component, but it will still use `config.previousSickLeavePeriod` from backend to be backwards compatible for now.